docs/cody/clients/model-configuration.mdx

```diff
@@ -4,223 +4,14 @@
 
 <Callout type="note">For Sourcegraph Cloud customers, configuring the available LLM models requires contacting your Sourcegraph account team representative</Callout>
 
-Sourcegraph v5.6.0 or later supports the ability to choose between different LLM models, allowing developers to use the best model for Cody Chat as needed.
+Cody Enterprise can be configured using one of two methods:
+1. Legacy "Completions" Configuration
+2. Model Configuration (Early Access Program)
 
-This is accomplished exposing much more flexible configuration options for Cody when using Sourcegraph Enterprise. The newer style of configuration is described next. However, you can still use the [Older style "Completions" Configuration](#legacy-completions-configuration).
+<Callout type="warning">Configuring Cody via your Sourcegraph Enterprise instance via the Legacy "Completions" Configuration site configuration section is going to be deprecated soon. For now, it will be backwards compatible and supported. The Model Configuration method is in Early Access Program and only avaiable on Sourcegraph v5.6.0 or later </Callout>
```

Comment:
Again, I don't think we should refer to this option as deprecated as long as its our primary recommendation.

Reply:
Agreed